### DISCUSSION TITLE
Welcome to OpenModelDB Discussions!

### DISCUSSION THREAD

**Discussion by RunDevelopment** [OpenModelDB/open-model-database#177] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and collect feedback. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

---

And don't forget to join the [Enhance Everything Discord server](https://discord.gg/enhance-everything-547949405949657098) to talk to even more members of the community. Not everyone is on GitHub after all :)

**Comment by Kuphi** [comment]
The invite for the Discord server is no longer working. Neither here nor on the website.

**Reply by joeyballentine** [reply]
this has been fixed
